crates/hyperdrive-math/src/long/close.rs

```diff
@@ -53,10 +53,54 @@ impl State {
 
         Ok(flat + curve)
     }
+
+    /// Calculates the market value in shares of a long position using the equation:
+    /// market_estimate = trading_proceeds - flat_fees_paid - curve_fees_paid
+    ///
+    /// trading_proceeds = flat_value + curve_value
```

Comment:
We have been preferentially rendering math with latex so it is more readable in the [docs page](https://docs.rs/hyperdrive-math/latest/hyperdrive_math/struct.State.html#method.open_long_curve_fee).

Reply:
Here's an example
```rs
    /// Calculates the market value in shares of a long position.
    ///
    /// market_estimate = trading_proceeds - flat_fees_paid - curve_fees_paid
    /// trading_proceeds = flat_value + curve_value
    /// ```math
    /// \begin{aligned}
    /// \text{trading_proceeds} &= \Delta y \cdot (1 - t) + \Delta y \cdot t \cdot p \\
    ///                  &= \Delta y \cdot ((1 - t) + (t \cdot p)) \\
    /// \text{flat_value} &= \Delta y \cdot (1 - t) \\
    /// \text{curve_bonds} &= \Delta y \cdot t \\
    /// \text{curve_value} &= \text{curve_bonds} \cdot p \\
    /// &= \Delta y \cdot t \cdot p \\
    /// \text{flat_fees_paid} &= \text{flat_value} \cdot \phi_f \\
    /// &= \Delta y \cdot (1 - t) \cdot flat_fee \\
    /// \text{curve_fees_paid} &= (\text{curve_bonds} - \text{curve_value}) \cdot \phi_c \\
    /// &= \Delta y \cdot t \cdot (1 - p) \cdot \phi_c \\
    /// \end{aligned}
    /// ```
    ///
    /// `$\Delta y = \text{bond_amount}$`
    /// `$p  = \text{spot_price}$`
    /// `$t  = \text{time_remaining}$`
    ```

Reply:
this is definitely harder to read in-code, which I think is worth considering going forward. We basically have to choose between easier-to-read in our online docs vs easier-to-read in our code. However, we should stick to this for now because that is what we do elsewhere and make the decision to change it later.

Reply:
Beautiful! Updated.